docs/adr/0002-resource-definition.md

```diff
@@ -24,17 +24,17 @@ In API Platform, this resource identifier is also named [IRI (Internationalized
 ```php
 <?php
 
-#[Resource]
+#[ApiResource]
 class Users
 {
-    #[ApiProperty(iri="hydra:member")]
+    #[ApiProperty(types=["hydra:member"])]
     public User[] $member = [];
```

Comment:
```suggestion
    public irable $member = [];
```

We should update the PHPDoc accordingly.

Reply:
maybe `iterable` :slightly_smiling_face:

Reply:
Fixed